src/spec-lang/expr_grammar.pegjs

```diff
@@ -1,6 +1,6 @@
 // Top-level rules
 Annotation
-    = StartingWhiteSpace annotation: (Invariant
+    = StartingWhiteSpace ('#' __)? annotation: (Invariant
```

Comment:
Same here - remove the '__' after '#'

Reply:
Addressed in https://github.com/ConsenSys/scribble/pull/44/commits/02b98f0d598cab8ac5fc53764fc058c3f88404a8